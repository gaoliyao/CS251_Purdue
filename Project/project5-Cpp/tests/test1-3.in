1
17 28
AAA AAB 1.10
AAB AAC 2.20
AAC AAD 3.30
AAD AAE 4.40
AAE AAF 5.50
AAF AAG 6.60
AAG AAH 7.70
AAH AAI 8.80
AAI AAJ 9.90
AAJ AAA 11.00
AAD AAA 11.00
AAB AAG 12.10
AAH AAD 13.20
AAG AAE 14.30
AAA AAH 15.40
AAJ AAD 16.50
AAB AAJ 17.60
AAB AAD 18.70
AAB AAH 19.80
AAA AAE 20.90
AAC BBD 43.97
BBD BBC 12.91
BBC CBD 10.10
CBD XYP 11.20
XYP XYA	14.21
XYA XYB 14.78
XYB XYC 26.90
XYC XYP 12.99
