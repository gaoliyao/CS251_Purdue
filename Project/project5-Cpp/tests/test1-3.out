1
4
AAC BBD
BBC BBD
BBC CBD
CBD XYP
